versions/baseline.json

```
@@ -4392,6 +4388,10 @@
       "baseline": "1.5",
       "port-version": 4
     },
+    "libfuse": {
+      "baseline": "3.16.2",
+      "port-version": 0
+    },
```

```suggestion
    "libfuse": {
      "baseline": "3.16.2",
      "port-version": 0
    },
```
Is this port newly added? Why can't I find it in the ports directory?

---

I do not know why that part of the file was modified. It was done by the tool. Let me try again.

---

This file now only modifies the section for the `robotraconteur` port. There may be a bug in x-add-version that caused this unrelated modification.

---

No, x-add-version does it right. But somebody add that port without using x-add-version, and neither CI nor reviewer noticed the glitch. And now it is in almost every PR, and reviewers are surprised... But normally it doesn't cause merge conflicts.